### PR TITLE
refactors MessageTimeGraph.py shifts to network.py

### PR DESCRIPTION
**Pull request by krishnacharya**
## What? Why?

Changes proposed in this pull request:
-adds aggregate analysis as well as day by day analysis to message time graph
-deletes MessageTimeGraph.py

## Checks
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure you have added required documentation
- [x] If you have refactored a code, make sure that you have compared the outputs pre and post refactoring. Add both pre and post refaoring screenshots below if the output is visual

## Any images?
day by day analysis before refactoring:
kubuntu-devel_2013_1_3_msg_time
day by day analysis after refactoring:
mtg3
aggregate analysis (for a few number of days):
mtgagg

## Notify reviewers
@rohangoel96

### CODE REVIEW

**Comment by krishnacharya** [comment]
I still have to change other files to improve the code climate grades but its underway, this was essential functionality which was missing.

**Comment by krishnacharya** [comment]
I have fixed code climate issues in local files as well as created new functions in util. However some files such as degreeNodeNumber , degreeMessageNumber are creating repetitions. Should i create a new function in util for these too?

File: lib/analysis/network.py [hunk 1]
```diff
@@ -468,13 +467,26 @@ def create_message_time_graph(log_dict, nicks, nick_same_list):
     G = util.to_graph(nick_same_list)
     conn_comp_list = list(connected_components(G))
 
+    def build_graphs(nick_sender,nick_receiver,line):
```
Comment (rohangoel96) [inline lib/analysis/network.py]: would it be better if we shift build_graphs and compare_nick to util? What do you think? They can be used in other modules as well.
moreover, as a novice user i will not know what exactly you mean by 'line' in the argument. change that?

**Comment by rohangoel96** [comment]
Yes, we do need the functions for them but i think @kaushik-rohit  is working on degreeNodeNumber & degreeMessageNumber. It'll be best if you collaborate with him.

File: sample.py [hunk 2]
```diff
@@ -11,13 +11,14 @@
...
+log_data = reader.linux_input(log_directory, channel_name, starting_date, ending_date)             
 nicks, nick_same_list = nickTracker.nick_tracker(log_data)
 
 
 # ============== ANALYSIS =============
-message_number_graph = network.message_number_graph(log_data, nicks, nick_same_list)
-degree_anal_message_numder = network.degree_analysis_on_graph(message_number_graph)
+#message_number_graph = network.message_number_graph(log_data, nicks, nick_same_list)
+#degree_anal_message_numder = network.degree_analysis_on_graph(message_number_graph)
+message_time_graph = network.create_message_time_graph(log_data, nicks, nick_same_list)
```
Comment (rohangoel96) [inline sample.py]: message_time_graph_list

**Pull request merged**